zebra-state/src/service/non_finalized_state/chain.rs

```diff
@@ -1521,6 +1521,20 @@ impl Chain {
                     sapling_shielded_data,
                     orchard_shielded_data,
                 ),
+                V6 {
+                    inputs,
+                    outputs,
+                    sapling_shielded_data,
+                    orchard_shielded_data,
+                    ..
+                } => (
+                    inputs,
+                    outputs,
+                    &None,
+                    &None,
+                    sapling_shielded_data,
+                    orchard_shielded_data,
```

Comment:
do we need to deal with issue bundle here?
(x2)

Reply:
Yeah, this is probably the best place to update the `issued_assets` map for the non-finalized part of the chain, same for `revert_chain_with()` when rolling back the chain for a fork.